airflow/utils/helpers.py

```diff
@@ -28,15 +28,8 @@
 import psutil
 from jinja2 import Template
 
+from airflow import AirflowException
```

Comment:
Shouldn't we use `from airflow. exceptions import AirflowException`

Reply:
Yeah. That is not yet enforced :). The change is in a separate PR to do so.